### PR TITLE
pycairo, bump version, fixes resolving symbol 'cairo_xlib_surface_get…

### PR DESCRIPTION
**Pull request by Begasus**
…_width'

### CODE REVIEW

**Comment by Begasus** [comment]
@OscarL didn't change anything recipe wise, this still holds python3.9, should we drop that?

File: dev-python/pycairo/pycairo-1.27.0.recipe [hunk 1]
```diff
@@ -6,7 +6,7 @@ COPYRIGHT="2023 pycairo developers"
 LICENSE="GNU LGPL v2.1"
 REVISION="4"
```
Comment (jmairboeck) [inline dev-python/pycairo/pycairo-1.27.0.recipe:7]:
```suggestion
REVISION="1"
```

Comment (Begasus) [reply dev-python/pycairo/pycairo-1.27.0.recipe]: Thanks, missed that one :)

**Comment by Begasus** [comment]
> Apparently, nothing on-tree actually requires pycairo? So moving it to 3.10 should be safe (assuming you use 3.10 for whatever you're testing pycairo).

OK 👍 was testing this for Meld when I saw the symbol error.

**Pull request merged**